src/vm/linux/types.rs

```diff
@@ -1 +1,75 @@
 // SPDX-License-Identifier: Apache-2.0
+
+pub const NR_CPUID_CONFIGS: usize = 12;
```

Comment:
Does it make sense to hardcode this value? We've already seen it change from 6 to 12. Can we guarantee it will stay at 12?